### PR TITLE
Adds rule about router dynamic segments being underscored.

### PR DESCRIPTION
**Pull request by michaeldupuisjr**

### CODE REVIEW

File: engineering/ember.md [hunk 1]
```diff
@@ -219,9 +219,22 @@ more complex logic in this case.
...
 ```
 
-## Routes
+## Routing
 
-### Perform all async actions required for the page to load in `model` hooks
+### Route naming
+Dynamic segments should be underscored. [This will allow Ember to resolve
+promises without extra serialization
+work](https://github.com/dockyard/shyft-rwe/pull/38#issuecomment-153872628).
```
Comment (danmcclain) [inline engineering/ember.md]: Link will die once the repository is transferred

Comment (michaeldupuisjr) [inline engineering/ember.md]: I guess someone should write a blog post to preserve your comment in the annals of history. ;-)

**Comment by drspaniel** [comment]
:+1:

**Comment by danmcclain** [comment]
@michaeldupuisjr There is a fancy twiddle that you can reference that some good samaritan put together for that purpose

**Comment by michaeldupuisjr** [comment]
@danmcclain Updated with a link to said Twiddle.

**Comment by danmcclain** [comment]
:+1:

**Comment by duggiefresh** [comment]
CHOO CHOO! MERGE TRAIN

**Pull request merged**